api/src/main/java/io/strimzi/api/kafka/model/status/Condition.java

```diff
@@ -71,8 +72,14 @@ public void setLastTransitionTime(String lastTransitionTime) {
         this.lastTransitionTime = lastTransitionTime;
     }
 
-    public void setAdditionalProperties(Map<String, Object> additionalProperties) {
-        this.additionalProperties = additionalProperties;
```

Comment:
Was this was this intended for deletion? I notice the additionalProperties field and getter are still in the file. Was this setter removed since setAdditionalProperty() method exists?

Reply:
Actually, I was more wondering why it was there in the first place. All the other classes we have use the `setAdditionalProperty` method which is also what the `UnknownPropertyPreserving` requires. So I think the `setAdditionalProperties` should not be needed. But maybe the tests will tell me differently :-o.